Update Korean trans

Lugggggia opened the pull request:
Fix minor error
->Edit GB world Fonts
->Edit Map0239.po (effect get line)
->Edit SR ending lines

Mimigris commented:
Be careful, you seem to have put some files to the root of the folder, make sure to move them to the Picture folder of the 2kki Korean translation!
![image](https://user-images.githubusercontent.com/108797333/196359424-4865c62c-4f97-4feb-a177-44526d3d83bd.png)

Lugggggia commented:
> Be careful, you seem to have put some files to the root of the folder, make sure to move them to the Picture folder of the 2kki Korean translation. ![image](https://user-images.githubusercontent.com/108797333/196359424-4865c62c-4f97-4feb-a177-44526d3d83bd.png) I've made a pull request on your branch to fix that and I've also edited GB pictures effects that you made to make sure they would not have display issues.

Thank you for the tip-off. I modified the file and added a few more translations.
-Multiple Ending Credit Translation
-Old version of Music Room Easter Egg Translation
I'm sorry, but before reflecting the PR you put in, I accidentally deleted some files, so I closed the PR for now (it's my mistake that I can't use GitHub well). I'm sorry.)
